Dockerfile.sdk

```diff
@@ -168,5 +168,13 @@ ARG TRITON_MODEL_ANALYZER_REPO_TAG
 ARG TRITON_MODEL_ANALYZER_REPO="https://github.com/triton-inference-server/model_analyzer@${TRITON_MODEL_ANALYZER_REPO_TAG}"
 RUN pip3 install "git+${TRITON_MODEL_ANALYZER_REPO}"
 
+# Entrypoint Banner
+ARG NVIDIA_TRITON_INFERENCE_SERVER_SDK_VERSION=21.10
+ARG NVIDIA_BUILD_ID=unknown
```

Comment:
Is the convention to use 'unknown' as the default build id?

Reply:
I took `unknown` as the default from our `build.py` convention: https://github.com/triton-inference-server/server/blob/400cc0cf7287de8f19c227442f1b98f5abe86e55/build.py#L959